config/config.py

```diff
@@ -1,9 +1,17 @@
-bind = "0.0.0.0:80"
+import os
+
+
+host = os.getenv("HOST") if os.getenv("HOST") else "0.0.0.0"
```

Comment:
Generally os.getenv got a built in a ability to set default so you can just have `os.getenv("HOST", "0.0.0.0")` instead of the if\else logic (it will take the value of the HOST envvar & if not set will return "0.0.0.0").

However Terraformize uses parse_it which I'll prefer to stick with (it gives users the ability to also use multiple config file types to configure variables instead of just envvars) so I think adding the needed values to https://github.com/naorlivne/terraformize/blob/master/terraformize/terraformize_configure.py#L4 & inheriting them from there will be a better option.

same goes for all the config variables set below that there might be a chance that the user will want to change

Reply:
Actually scratch that, the gunicron config will get called before the parse_it parser so rather then having 2 parse_it parsers running we should just stick to os.getenv("HOST", "0.0.0.0") formatting with sane defaults as the needed changes.